pkg/bindings/play/play.go

```diff
@@ -2,16 +2,45 @@ package play
 
 import (
 	"context"
+	"io/ioutil"
 	"net/http"
 	"net/url"
 	"os"
 	"strconv"
 
 	"github.com/containers/image/v5/types"
 	"github.com/containers/libpod/pkg/bindings"
+	authBindings "github.com/containers/libpod/pkg/bindings/auth"
 	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
```

Comment:
How much does this grow the remote client?

Reply:
This would effectively revert the changes from @baude and rebump from 33M to 41M.

Reply:
I want to go back to the initial approach of sending all credentials over to the server (see https://github.com/containers/image/pull/942#issue-422757846). The amount of work and complexity is getting out of hand.

Reply:
This would avoid repulling the k8s stuff back in :)